src/pyfaf/opsys/fedora.py

```diff
@@ -220,74 +218,68 @@ def save_ureport(self, db, db_report, ureport, packages, flush=False, count=1):
 
     def get_releases(self):
         result = {}
-        collections = self._pkgdb.get_collections()["collections"]
+        url = "https://pdc.fedoraproject.org/rest_api/v1/releases/"
```

Comment:
I would love to see this as config option.

Reply:
good point. Would it be better to put it in config as pdc_release_url or pdc_url? (It will pull releases and components from pdc, if ti should be two urls in config or one and add to it in code)